cypress/e2e/models/migration/migration-waves/migration-wave.ts

```diff
@@ -136,10 +137,14 @@ export class MigrationWave {
         MigrationWave.open();
         this.expandActionsMenu();
         cy.contains(manageApplications).click();
-        selectItemsPerPage(100);
+        cy.get(modal)
```

Comment:
This is OK, but I wanted to create a function something like `callWithin` that will accept function as a parameter and call existing function inside of particular HTML code only.

Reply:
its a good idea, I'll add it!

Reply:
Solved in db9eb6eb0ced097ccd0779672de1d5239f220d46.